### ISSUE TITLE
Corewar-ui tests do not execute and terminate

### ISSUE THREAD

**Issue by gareththegeek** [corewar/corewar.io#51]
`lerna run test --scope "{corewar-api,corewar}"`
executes and terminates correctly
`lerna run test --scope "{corewar-api,corewar,corewar-ui}"`
does not.   Instead it hangs and then when terminated states:
>corewar-ui: > react-scripts test --env=jsdom
corewar-ui: No tests found related to files changed since last commit.

This is blocking the CI build.  It seems that the `--onlyChanged` or `--watch` or both are provided to `jest` by `react-scripts` by default perhaps?

**Comment by dougajmcdonald** [comment]
You'd want to put the jest 'ci' flag on if you wanna run the tests in the ci build I think as it makes it Jon interactive? Odd that this runs ok in other environments though

**Comment by gareththegeek** [comment]
Yeah, I think it's actually ok on the CI build.  The problem I'm getting now on the CI build is a few test failures:

```
corewar-ui: FAIL src/tests/simulator/instruction.test.js
corewar-ui:   ● Test suite failed to run
corewar-ui:     Cannot find module './../../helpers/coreEnums' from 'instruction.test.js'
corewar-ui:        5 | 
corewar-ui:        6 | import Instruction from './../../features/simulator/instruction'
corewar-ui:     >  7 | import { ModeType } from './../../helpers/coreEnums'
corewar-ui:          | ^
corewar-ui:        8 | 
corewar-ui:        9 | it('renders without crashing', () => {
corewar-ui:       10 |   const props = {
corewar-ui:       at Resolver.resolveModule (../../node_modules/jest-resolve/build/index.js:259:17)
corewar-ui:       at Object.<anonymous> (src/tests/simulator/instruction.test.js:7:1)
```

See this build for example: https://github.com/corewar/corewar.io/pull/46/checks?check_run_id=471049745

**Comment by gareththegeek** [comment]
Can reproduce this issue locally with `set ci=true && lerna run test --scope corewar-ui --stream`

**Comment by gareththegeek** [comment]
Test failure was due to breaking changes - resolved by #46
